litmus-portal/graphql-server/graph/schema.graphqls

```diff
@@ -104,6 +104,18 @@ input ChaosWorkFlowInput {
   cluster_id: ID!
 }
 
+input ChaosWorkflowUpdateInput {
```

Comment:
Reuse ChoasWorkflowInput by adding an optional param in it for `workflow_id` this will only be used in the update workflow case and not in create workflow.

Reply:
Yes, it can be reuse